deployment/bin/start-condor.sh

```diff
@@ -16,4 +16,13 @@ if [ "$SET_NOBODY_USER_UID" ] ; then
     usermod -u "$SET_NOBODY_USER_UID" nobody -o
 fi
 
-exec $(condor_config_val MASTER) -f -t 2>&1 
+if [ "$CONDOR_SUBMIT_WORKDIR" ] ; then
+    mkdir -p $CONDOR_SUBMIT_WORKDIR
+    chmod 01777 $CONDOR_SUBMIT_WORKDIR
+else
+    mkdir -p /mnt/awe/condor/condor_job_execute
+    chmod 01777 /mnt/awe/condor/condor_job_execute
```

Comment:
This looks wrong, I'd expect it to be chmod 1777.  But it seems like 01777 has the same effect.

Reply:
Do you want me to change it? It's what I read somewhere while searching for the condor issue.

Reply:
Hmm.  Let's leave it here since we know it works, and in a future PR to develop we can change it to 1777.

Reply:
https://www.google.com/search?q=htcondor+01777&oq=htcondor+01777&aqs=chrome..69i57.1520j0j4&sourceid=chrome&ie=UTF-8